viewer/src/public/RevealManager.ts

```diff
@@ -51,6 +53,15 @@ export class RevealManager<TModelIdentifier> {
     this._pointCloudManager = pointCloudManager;
     this._effectRenderManager = new EffectRenderManager(this._cadManager.materialManager);
     this.initLoadingStateObserver(this._cadManager, this._pointCloudManager);
+    this._updateSubject = new Subject();
+    this._updateSubject
+      .pipe(
+        auditTime(5000),
+        tap(() => {
+          trackNavigation({ moduleName: 'RevealManager', methodName: 'update' });
```

Comment:
It’s a bit confusing that we send method name update which doesn’t exist. I suggest to create that method and put tracking updates there.

Reply:
Technically it is tracking update, but throttled every 5000ms. Update is a function further down where _updateSubject.next() is called.

Reply:
Yeah, it's a bit confusing - but the code sets up a RxJS issue that is _triggered_ from `update()` so the event comes from the `update`-function. Makes sense?